apps/settings/elements/firefox_accounts.html

```diff
@@ -29,14 +29,14 @@ <h2 data-l10n-id="fxa-confirm-account">Confirm Your Account</h2>
       </header>
       <ul>
         <li class="description">
-          <p data-l10n-id="fxa-verification-email-sent-msg" id="fxa-unverified-text">
+          <p id="fxa-unverified-text">
             We will send an email to: foo@bar.com
           </p>
           <p data-l10n-id="fxa-please-verify">
             Click the verification link in your email to confirm your account.
           </p>
-          <p id="fxa-resend-email">
-            Don't see an email? <a href="#">Resend</a>.
+          <p id="fxa-resend-email" data-l10n-id="fxa-dont-see-email-resend">
+            Don't see an email? <a href="#" id="fxa-resend">Resend</a>.
```

Comment:
Absolutely. It will never show up because we pretranslate to default language during build time, so any strings defined in HTML will be overwrote. We can as well remove them :)